tests/foreman/cli/test_contentaccess.py

```diff
@@ -15,251 +17,153 @@
-            2. install a package, that will need errata to be applied
-            3. list the host applicable errata with searching the required
-               errata id
-
-        :expectedresults: errata listed successfuly and is installable
-
-        :BZ: 1344049, 1498158
-
-        :CaseImportance: Critical
-        """
-        with VirtualMachine(distro=DISTRO_RHEL7) as vm:
-            self._setup_virtual_machine(vm)
-            # install the packages that require updates
-            result = vm.run(f'yum install -y {REAL_RHEL7_0_0_PACKAGE}')
-            self.assertEqual(result.return_code, 0)
-            result = vm.run(f'rpm -q {REAL_RHEL7_0_0_PACKAGE}')
-            self.assertEqual(result.return_code, 0)
-            # check that package errata is applicable
-            for _ in range(30):
-                erratum = Host.errata_list(
-                    {'host': vm.hostname, 'search': f'id = {REAL_RHEL7_0_ERRATA_ID}'}
-                )
-                if erratum:
-                    break
-                time.sleep(10)
-            self.assertEqual(len(erratum), 1)
-            self.assertEqual(erratum[0]['installable'], 'true')
-
-    @pytest.mark.tier2
-    def test_negative_rct_not_shows_golden_ticket_enabled(self):
-        """Assert restricted manifest has no Golden Ticket enabled .
-
-        :id: 754c1be7-468e-4795-bcf9-258a38f3418b
-
-        :steps:
-
-            1. Run `rct cat-manifest /tmp/restricted_manifest.zip`.
-
-        :CaseAutomation: Automated
-
-        :expectedresults:
-            1. Assert `Content Access Mode: org_environment` is not present.
-
-        :CaseImportance: High
-        """
-        org = make_org()
-        # upload organization manifest with org environment access enabled
-        manifest = manifests.clone()
-        manifests.upload_manifest_locked(org['id'], manifest, interface=manifests.INTERFACE_CLI)
-        result = ssh.command(f'rct cat-manifest {manifest.filename}')
-        self.assertEqual(result.return_code, 0)
-        self.assertNotIn('Content Access Mode: org_environment', '\n'.join(result.stdout))
-
-    @pytest.mark.tier2
-    @pytest.mark.upgrade
-    def test_positive_rct_shows_golden_ticket_enabled(self):
-        """Assert unrestricted manifest has Golden Ticket enabled .
-
-        :id: 0c6e2f88-1a86-4417-9248-d7bd20584197
-
-        :steps:
-
-            1. Run `rct cat-manifest /tmp/unrestricted_manifest.zip`.
-
-        :CaseAutomation: Automated
-
-        :expectedresults:
-            1. Assert `Content Access Mode: org_environment` is present.
-
-        :CaseImportance: Medium
-        """
-        result = ssh.command(f'rct cat-manifest {self.manifest.filename}')
-        self.assertEqual(result.return_code, 0)
-        self.assertIn('Content Access Mode: org_environment', '\n'.join(result.stdout))
+    :CaseImportance: Medium
+    """
+    result = ssh.command(f'rct cat-manifest {module_gt_manifest.filename}')
```

Comment:
this could be done with a newly cloned gt manifest, freeing up the module_gt_manifest fixture to return the organization.

Reply:
@JacobCallahan many changes made, please review once more. Thank you